presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkModule.java

```diff
@@ -285,6 +287,8 @@ protected void setup(Binder binder)
         jsonCodecBinder(binder).bindJsonCodec(PrestoSparkLocalShuffleReadInfo.class);
         jsonCodecBinder(binder).bindJsonCodec(PrestoSparkLocalShuffleWriteInfo.class);
         jsonCodecBinder(binder).bindJsonCodec(BatchTaskUpdateRequest.class);
+        jsonCodecBinder(binder).bindJsonCodec(BroadcastFileInfo.class);
```

Comment:
As suggested in another PR: https://github.com/prestodb/presto/pull/19820#discussion_r1259581771

Please see if you can merge all these codec into 1 collection and inject using mutli-bind.

Reply:
Can you please clarify what multi-bind means ?

Reply:
[multi-bind](https://google.github.io/guice/api-docs/3.0/javadoc/com/google/inject/multibindings/Multibinder.html). You can bind collections like List, Map, Set.

Reply:
We can address this refactoring in followup PR